app/upgrades/v4/constants.go

```diff
@@ -0,0 +1,7 @@
+package v4
+
+const (
+	// UpgradeName defines the on-chain upgrade name.
+	UpgradeName         = "v4-Commission"
```

Comment:
This should be coupled with sdk47 upgrade

Reply:
yes, we'll change in sdk47 PR